wremnants/plot_tools.py

```diff
@@ -1601,6 +1601,11 @@ def write_index_and_log(
             + "\n"
         )
         logf.write(meta_info)
+        meta_info = narf.ioutils.make_meta_info_dict(
+            "notebooks", args=args, wd=common.base_dir
+        )
+        logf.write(f"git hash: {meta_info['git_hash']}\n")
+        logf.write(f"git diff: {meta_info['git_diff']}\n")
```

Comment:
Is this needed? Now the git hash and diff appear twice in the log files, once from here and another time from the meta info.

Reply:
I added this because the summary plots had absolutely no info in them. But it's true that it could duplicate the info in the hist file. But in principle the diffs don't have to be the same, right? Maybe it should be added but in a more subtle way

Reply:
Good point, yes OK I agree to keep it